estate/data/estate_tour.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
```

Comment:
User probably don't want this tour to show up anytime they upgrade, so setting this as noupdate seems appropriate